### DISCUSSION TITLE
Problem linking EBSD and DIC

### DISCUSSION THREAD

**Discussion by ZheningYang1** [MechMicroMan/DefDAP#109] [General]
I am analysing DIC of Inconel 718. When I run 
dicMap.plotMaxShear(vmin=0, vmax=0.4, plotScaleBar=True, plotGBs=True)

It report error: 
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[44], line 1
----> 1 dicMap.plotMaxShear(vmin=0, vmax=0.4, plotScaleBar=True, plotGBs=True)

File ~\Anaconda3\envs\defdap2\lib\site-packages\defdap\hrdic.py:796, in Map.plotMaxShear(self, **kwargs)
    791 params = {
    792     'clabel': 'Effective Shear Strain'
    793 }
    794 params.update(kwargs)
--> 796 plot = self.plotMap('eMaxShear', **params)
    798 return plot

File ~\Anaconda3\envs\defdap2\lib\site-packages\defdap\hrdic.py:772, in Map.plotMap(self, component, **kwargs)
    766 plotParams = {
    767     'plotColourBar': True,
    768     'clabel': component
    769 }
    770 plotParams.update(kwargs)
--> 772 plot = MapPlot.create(self, self.crop(self.component[component]), **plotParams)
    774 return plot

File ~\Anaconda3\envs\defdap2\lib\site-packages\defdap\plotting.py:675, in MapPlot.create(cls, callingMap, mapData, fig, figParams, ax, axParams, plot, makeInteractive, plotColourBar, vmin, vmax, cmap, clabel, plotGBs, dilateBoundaries, boundaryColour, plotScaleBar, scale, highlightGrains, highlightColours, highlightAlpha, **kwargs)
    672     plot.addColourBar(clabel)
    674 if plotGBs:
--> 675     plot.addGrainBoundaries(
    676         colour=boundaryColour, dilate=dilateBoundaries, kind=plotGBs
    677     )
    679 if highlightGrains is not None:
    680     plot.addGrainHighlights(
    681         highlightGrains,
    682         grainColours=highlightColours, alpha=highlightAlpha
    683     )

File ~\Anaconda3\envs\defdap2\lib\site-packages\defdap\plotting.py:440, in MapPlot.addGrainBoundaries(self, kind, boundaries, colour, dilate, draw, **kwargs)
    438 boundariesImage = boundaries
    439 if boundariesImage is None:
--> 440     boundariesImage = self.callingMap.boundaries
    441 boundariesImage = -boundariesImage
    443 if dilate:

File ~\Anaconda3\envs\defdap2\lib\site-packages\defdap\hrdic.py:586, in Map.boundaries(self)
    583 boundaries = boundaries > 0.1
    585 boundaries = mph.skeletonize(boundaries)
--> 586 mph.remove_small_objects(boundaries, min_size=10, in_place=True,
    587                          connectivity=2)
    589 # crop image if it is a simple affine transform
    590 if type(self.ebsdTransform) is tf.AffineTransform:
    591     # need to apply the translation of ebsd transform and
    592     # remove 5% border

TypeError: remove_small_objects() got an unexpected keyword argument 'in_place'

How can I solve this problem? Thanks.

**Comment by rhysgt** [comment]
Hi Zhening. This problem was fixed, so please try downloading the latest version of the master branch.

**Reply by ZheningYang1** [reply]
thanks for your help Rhys.
we just updated to the most current version but it says i already have that version 
DefDAP               0.93.5
what is the most current version ?
